ophyd/runengine/runengine.py

```diff
@@ -218,19 +235,22 @@ def _start_scan(self, **kwargs):
             # TODO: timestamp this datapoint?
             # data.update({'timestamp': time.time()})
             # pass data onto Demuxer for distribution
-            print('datapoint[{}]: {}'.format(seq_num, detvals))
+            self._sessionmgr._logger.info(
+                '{}'.format(self._demunge_values(detvals, names)))
             # grab the current time as a timestamp that describes when the
             # event data was bundled together
             bundle_time = time.time()
             # actually insert the event into metadataStore
             try:
-                print('\n\ninserting event {}\n------------------'.format(seq_num))
+                self._sessionmgr._logger.debug(
+                    'inserting event {}------------------'.format(seq_num))
```

Comment:
Loggers are one case where using the old-style formatting still makes sense.  If you do

```python
logger.debug('%s foo bar %d", 'str', 5)
```

The formatting will only be done if the message is actually propagated out to a handler.